karma.conf.js

```diff
@@ -11,7 +11,8 @@ module.exports = function(config) {
     plugins: [
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
-      require('karma-jasmine-html-reporter'),
```

Comment:
Don't we still need the html reporter for the output in the browser (This is the output that i use)?

Reply:
Well, I thought no one is using this reporter.
I'll add it right back.